src/shared/dates.ts

```diff
@@ -16,6 +16,13 @@ export function getHackathonEndDate(): moment.Moment {
   return moment('2019-01-20');
 }
 
+/**
+ * Returns the datetime when applications close.
+ */
+export function getAdvertisedApplicationsEnd(): moment.Moment {
```

Comment:
Could we update the docstring to something like "Returns the datetime applications are advertised to close.  This is the closing date displayed to hackers.  Applications won't actually close until `getApplicationsEnd`, at which point links to start applications will be removed."?

Reply:
We could also update the `getApplicationsEnd` docstring to make it clear this date is when we'll remove links to apply.

Reply:
Good ideas, I'll add this in